stdlib/public/Concurrency/Actor.cpp

```diff
@@ -1518,7 +1432,8 @@ void DefaultActorImpl::destroy() {
   // Tasks on an actor are supposed to keep the actor alive until they start
   // running and we can only get here if ref count of the object = 0 which means
   // there should be no more tasks enqueued on the actor.
-  assert(!oldState.getFirstJob() && "actor has queued jobs at destruction");
+  // TODO: It is not safe to access prioritisedJobs here, right?
+  assert(!oldState.getFirstUnprioritisedJob() && "actor has queued jobs at destruction");
 
   if (oldState.isIdle()) {
       return;
```

Comment:
I think it is possible to assert that `prioritisedJobs` are empty here, if `oldState` is loaded using `std::memory_order_acquire`